subsystems/viamserver/viamserver.go

```diff
@@ -171,7 +171,7 @@ func (s *viamServer) Stop(ctx context.Context) error {
 		s.logger.Error(err)
 	}
 
-	if s.waitForExit(ctx, stopTimeout/2) {
+	if s.waitForExit(ctx, stopTimeout) {
```

Comment:
I think this is the wrong line to modify. The timeout above this one (line 163) is the wait before sending KILL. That's probably the one to modify. This one is just waiting for everything to be dead which SHOULD be only second or so at most, as kill is kernel-only, just yanks the rug. No chance for the program to delay things on its own.

Reply:
sorry! i literally got confused on lines because they looked almost the same